stream/fuzz/fuzz_targets/connection.rs

```diff
@@ -40,7 +40,7 @@ impl<'a> arbitrary::Arbitrary<'a> for FuzzInput {
             .collect::<Result<Vec<_>, _>>()?;
 
         // Generate constrained parameters
-        let max_message_size = u.int_in_range(177..=1024 * 1023)?;
```

Comment:
Are you able to document why these numbers are chosen, especially the lower-bound here?

Reply:
Pushed a comment to that effect ; we could calculate such a bound dynamically, but that would require generating a public key on every fuzz input, which seems a bit silly.